### ISSUE TITLE
(FED) Maps - Page jumping to top after closing full screen map view

### ISSUE THREAD

**Issue by michaelasemple** [visitscotland/vs-component-library#116]
### Description
The page scrolls up to the top once you close or escape the full screen view on map (vertical and horizontal) – all iPads

### To Reproduce

Steps to reproduce:

1. Navigate to https://feature.visitscotland.com/?vs_brxm_host=172.28.87.25&vs_brxm_port=8030&vs_brxm_http_host=feature.visitscotland.com&vs_ssr_http_port=8230&vs_tln_http_port=8330&vs_feature_branch=PR-56&vs_ssr_proxy=on
https://feature.visitscotland.com/test/examples/outlander 
2. Scroll to the map
3. Open full view
4. Close the full view using the x icon or the full view icon or press esc

### Expected behaviour

The page shouldn't scroll, user should see the map.

### Device and version
All iPads

### Additional context
was raised as a bug during testing of the outlander map: https://jira.visitscotland.com/browse/VSD-562

**Comment by duncansanderson** [comment]
I don't think this is a bug with our code, I think it's an issue with full screen mode on iOS devices.

I have tested a [Mapbox full screen example](https://docs.mapbox.com/mapbox-gl-js/example/fullscreen/) which has the same behaviour when exiting a full screen map on iPad. This would suggest that the issue isn't with our code. I have also created a [separate example](https://codepen.io/duncansa/full/MWRaEQX), not using a map, of entering and exiting full screen (buttons at the bottom of the page), which also has the same issue. This would bring me to the conclusion that the problem is caused by iOS devices rather an a bug our code or the map code.

@MichaelGrahamVS are you happy for this ticket to be closed since we can't do anything about it?
